cmd/testbucket/grammar_test.go

```
@@ -0,0 +1,494 @@
+}
+
+func fixtureShard(i int) Unit {
+	p := livePkg("bamlutils/llmhttp", "bamlutils", modeWork, true)
+	return Unit{
+		ID: countShardID(p.ImportPath, i), Kind: kindCountShard, Seconds: 150,
+		Packages: []LivePackage{p}, Module: p.Module, Mode: p.Mode,
+		Count: 17, Shard: i, Shards: 6,
+	}
+}
+
+func fixtureSlice() Unit {
```

golangci-lint (2.12.2)

[error] func fixtureSlice is unused

(unused)